server/api/store.go

```diff
@@ -130,6 +131,22 @@ func newStoreHandler(svr *server.Server, rd *render.Render) *storeHandler {
 	}
 }
 
+func (h *storeHandler) errorResp(w http.ResponseWriter, err error) {
+	if err == nil {
+		// TODO: log this condition
+		h.rd.JSON(w, http.StatusInternalServerError, "nil error")
+		return
+	}
+	if errCode, ok := errors.Cause(err).(errcode.ErrorCode); ok {
+		w.Header().Set("Tidb-Error-Code", string(errCode.Code()))
```

Comment:
s/Tidb/TiDB/

Reply:
I am trying to follow HTTP convention where normally headers are written so that the first letter is upper case, but I guess we can change it since headers are case insensitive.

My goal was to have the TiDB ecosystem use the same header. I use `TiDB` as a term for the entire ecosystem, and I would prefer to use `TiDB SQL` for the SQL layer to avoid confusion.
Now that `Code()` is already a string, it can be prefixed with `pd` if we want, so it could be `pd.store.state.tombstoned`. However, my thought was to distinguish by subject matter rather than by service. TiKV could also give back the same `store.state.tombstoned` errror, which I think is a good thing. The main problem would be if there are two services that use the same term for different subjects. But that is already an issue that should be addressed.